addons/project/static/src/components/subtask_kanban_list/subtask_kanban_list.js

```diff
@@ -43,70 +44,38 @@ export class SubtaskKanbanList extends Component {
         });
     }
 
-    get fieldNames() {
-        return Object.keys(this.fields);
-    }
-
-    get fields() {
-        const { display_name, state, user_ids, project_id } = this.props.record.fields;
-        return {
-            display_name,
-            state,
-            user_ids,
-            project_id,
-        };
+    async onSubTaskCreated(ev) {
+        this.subtaskCreate.open = true;
     }
 
-    get activeFields() {
-        return {
-            display_name: {},
-            state: {
-                name: "state",
-                viewType: "kanban",
-                field: getFieldFromRegistry(this.fields.state.type, "project_task_state_selection", "kanban"),
-            },
-            user_ids: {
-                name: "user_ids",
-                field: getFieldFromRegistry(this.fields.user_ids.type, "many2many_avatar_user", "kanban"),
-            },
-            project_id: {
-                field: getFieldFromRegistry(this.fields.project_id.type, "project", "kanban")
-            },
-        };
+    async _onBlur() {
+        this.subtaskCreate.open = false;
     }
 
-    async onSubTaskSaved(subTask) {
-        const ids = this.subTasksRead.map((t) => t.id);
-        this.subTasksRead = await this.orm.searchRead(
-            this.props.record.resModel,
-            [["id", "in", ids]],
-            this.fieldNames
-        );
-        const isKnownAsClosed = this.subTaskClosed.has(subTask.resId);
-        const isClosed = subTask.data.state.startsWith("1_");
-        if (isKnownAsClosed && !isClosed) {
-            this.subTaskClosed.delete(subTask.resId);
-        } else if (!isKnownAsClosed && isClosed) {
-            this.subTaskClosed.add(subTask.resId);
-        } else { // nothing to do
-            return;
-        }
-        await this.props.record.load();
+    async _onSubtaskCreateNameChanged(name) {
+        await this.orm.create("project.task", [{ name: name, parent_id: this.props.record.resId, user_ids: this.props.record.data.user_ids.resIds }]);
```

Comment:
```suggestion
        await this.orm.create("project.task", [{ display_name: name, parent_id: this.props.record.resId, user_ids: this.props.record.data.user_ids.resIds }]);
```
to be able to use the shortcuts of Raouf

Reply:
:+1: